docs/general/manage-tokens/README.mdx

```diff
@@ -1,30 +1,201 @@
+| Cipher    | `oasis1`           | ed25519, sr25519            | <ul><li>Oasis CLI</li></ul> |
+| Emerald   | `0x`, `oasis1` | secp256k1, ed25519, sr25519 | <ul><li>Metamask and other EVM-compatible wallets (transfers only)</li><li>Oasis Wallet - Browser Extension</li><li>Oasis Wallet - Web (deposits and withdrawals only)</li><li>Oasis CLI</li></ul> |
+
+[Bech-32 encoding]: https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki#bech32
+
+## Obtain ROSE
+
+The most common way to obtain ROSE is by buying it on a centralized
+[cryptocurrency exchange]. At time of writing this chapter, all exchanges
+operated solely on the **consensus layer**. This means that you can only move
+the tokens in or out of the exchange using **your consensus account**.
+
+An alternative way to obtain ROSE is by swapping another cryptocurrency with
+ROSE on an on-chain exchange also known as **the [decentralized exchange] (DEX)**.
+At time of writing at least one such exchange was operating on Sapphire. In this
+case, your originating token is sent to the contract address on the source chain
+and the payout is made to **your account on Sapphire**.
+
+[cryptocurrency exchange]: https://en.wikipedia.org/wiki/Cryptocurrency_exchange
+[decentralized exchange]: https://en.wikipedia.org/wiki/Decentralized_finance#Decentralized_exchanges
+
+## Check your account
+
+To check the balance of your consensus account, you can use the
+[Oasis Scan](https://www.oasisscan.com) block explorer. Enter your `oasis1`
+address at the top and hit enter. For example:
+
+![Account details of entered oasis1 address in Oasis Scan](../images/manage-tokens/oasisscan_account_details.png)
+
+The "Amount" is a sum of three values:
+
+- the "Available" tokens that can immediately be transferred,
+- the "Escrow" tokens that are delegated,
+- the "Reclaim" tokens that are waiting for the debonding period to pass.
+
+To check the account's deposits and withdrawals navigate to "Transactions" pane
+and press "ParaTime" on the right side, next to the "Consensus" button.
+You will see all ParaTime-related transactions including deposits, withdrawals,
+transfers and even smart contract transactions.
+
+![Search result of oasis1 address - Account details](../images/manage-tokens/oasisscan_paratime_txes.png)
+
+Furthermore, you can view the transaction details, if you click on a
+transaction's "Tx Hash". Among others, you will see the transaction type, the
+"from", "to" and "amount" fields.
+
+![Tx Hash - Transaction details](../images/manage-tokens/oasisscan_paratime_tx_details.png)
+
+:::info
+
+Be aware that the [Oasis Scan Blockchain Explorer](https://www.oasisscan.com)
+is built for consensus layer. If you want to explore Sapphire (0x addresses,
+Token Transfers, Contract Calls, etc.), you have to use the
+[Sapphire Blockchain Explorer](https://explorer.oasis.io/mainnet/sapphire).
 
 :::
 
-## Quick Navigation
+## See Also
```

Comment:
Actually the whole See Also section can be removed, because those topics were already covered inside the chapter. The See Also usually refers to similar chapters in other parts (e.g. from the core or node operators).